Source/WebCore/Modules/WebGPU/GPUTexture.cpp

```diff
@@ -61,11 +66,29 @@ static PAL::WebGPU::TextureViewDescriptor convertToBacking(const std::optional<G
 
 Ref<GPUTextureView> GPUTexture::createView(const std::optional<GPUTextureViewDescriptor>& textureViewDescriptor) const
 {
-    return GPUTextureView::create(m_backing->createView(convertToBacking(textureViewDescriptor)));
+    auto matchesKey = [&textureViewDescriptor] (auto& kvp) {
+        return kvp.first == textureViewDescriptor;
+    };
+    if (auto index = m_cachedTextureViews.findIf(matchesKey); index != notFound)
+        return m_cachedTextureViews[index].second;
+
+    auto textureView = GPUTextureView::create(m_backing->createView(convertToBacking(textureViewDescriptor)));
+    m_cachedTextureViews.append(std::make_pair(textureViewDescriptor, textureView));
+
+    return textureView;
```

Comment:
Can we do this in a follow-up patch? Implementing a texture view cache seems different than "no objects are getting destroyed ever"